src/framework/components/light/component.js

```diff
@@ -334,6 +339,12 @@ function _defineProps() {
     _defineProperty("shadowBias", 0.05, function (newValue, oldValue) {
         this.light.shadowBias = -0.01 * newValue;
     });
+    _defineProperty("shadowCascades", 1, function (newValue, oldValue) {
+        this.light.numCascades = newValue;
```

Comment:
It is worth some sanitizing of the parameter here? Or a DEBUG message when it's not 1, 2, 3 or 4?

Reply:
good idea and done, thanks